README.md

```diff
@@ -65,6 +65,7 @@ Parameters (specified as one object passed into hot-shots):
 * `udsGracefulErrorHandling`: Used only when the protocol is `uds`. Boolean indicating whether to handle socket errors gracefully. Defaults to true.
 * `udsGracefulRestartRateLimit`: Used only when the protocol is `uds`. Time (ms) between re-creating the socket. Defaults to `1000`.
 * `closingFlushInterval`: Before closing, StatsD will check for inflight messages. Time (ms) between each check. Defaults to `50`.
+* `udpSocketOptions`: Specify these options to control how StatsD will create a UDP (dgram) socket.
```

Comment:
I'm going to tweak the wording of this a little bit, to make clear for those who don't know about it what "these" refers to.  But I can do that myself after merging, so I can just get this all set now.